src/internal/toHexPrice.ts

```diff
@@ -0,0 +1,20 @@
+import { BigIntish } from '@brinkninja/types'
+
+const DECIMALS = 18;
+
+export function convertToHexPrice (tokenOutAmount: BigIntish, tokenInAmount: BigIntish): string {
```

Comment:
I feel like we should split this into more lower level fns:

`tokenAmountsToPrice(amount0, amount1): number` to get a price number from two bigintish token amounts
`priceToX96(price): bigint` to convert from a price with decimal (like 12.34) to X96 bigint
`bigintToHex(bn): string` to convert any BigInt to hex string

Reply:
done